coretex/cli/config.py

```diff
@@ -0,0 +1,151 @@
+from typing import Optional, Tuple
+from pathlib import Path
+
+from tabulate import tabulate
+
+import click
+
+from .utils import arrowPrompt
+from ..networking import networkManager
+from ..statistics import getAvailableRamMemory
+from ..configuration import loadConfig, saveConfig, isUserConfigured, isNodeConfigured
+
+
+def authenticate(retryCount: int = 0) -> Tuple[str, str, str, str]:
+    if retryCount >= 3:
+        raise click.ClickException("") #raise errror
+
+    username = click.prompt("Email", type = str)
+    password = click.prompt("Password", type = str, hide_input = True)
+
+    response = networkManager.authenticate(username, password, False)
+
+    if response.hasFailed():
+        click.echo("Failed to authenticate. Please try again...")
+        return authenticate(retryCount + 1)
+
+    jsonResponse = response.getJson(dict)
+
+    return username, password, jsonResponse["token"], jsonResponse["refreshToken"]
+
+
+def registerNode(name: str) -> Optional[str]:
+    params = {
+        "machine_name": name
+    }
+    response = networkManager.post('service', params = params)
```

Comment:
Use double quotes in Python. There is no need to pass `params` as a keyword parameter, it can be safely passed as a positional parameter.